<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HighLevel Behavior Grid Master! :p</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>541</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>148</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_id">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_x">
       <property name="minimum">
        <number>-550</number>
       </property>
       <property name="maximum">
        <number>550</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_y">
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Center:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_cx">
       <property name="minimum">
        <number>-550</number>
       </property>
       <property name="maximum">
        <number>550</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_cy">
       <property name="minimum">
        <number>-400</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Parent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="b_par">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="b_name"/>
     </item>
     <item>
      <widget class="QPushButton" name="update">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirrorX">
       <property name="text">
        <string>Mirror-X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mirrorY">
       <property name="text">
        <string>Mirror-Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
